app/src/debug/res/values/strings.xml

```diff
@@ -103,6 +103,7 @@
     <string name="menu_setting_title">Settings</string>
     <string name="menu_setting_friends">Friends</string>
     <string name="menu_action_settings">Settings</string>
+    <string name="menu_action_clear_map">Clear Map</string>
```

Comment:
why add both debug string and main string. you can add only one to main/strings.xml